### PR TITLE
[DOCS-6337] CCM Updating Azure Setup Doc 

### PR DESCRIPTION
**Pull request by klara-kr**
### What does this PR do? What is the motivation?

This PR attemps to clarify the CCM azure setup documentation, as many customer still trouble to setup the product easily.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by crousseaux — APPROVED** [review]
lgtm

File: content/en/cloud_cost_management/aws.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+| `is_aws_ec2_capacity_reservation`| Whether the usage is associated with a Capacity Reservation.|
+| `is_aws_ec2_spot_instance`   | Whether the usage is associated with a Spot Instance.|
+| `is_aws_ec2_savings_plan`    | Whether the usage is associated with a Savings Plan.|
+
+## Billing conductor
+Billing conductor enables you to simplify your bill by customizing the billing rates, distributing credits and fees, and sharing overhead costs at your discretion. You can also select which accounts to include in the CUR.
+
+To create a billing conductor CUR, follow the [AWS Cost and Usage Reports user guide][8]. Ensure the CUR meets [Datadog's requirements][9].
+After the billing conductor CUR is created, follow the Cloud Cost Management instructions above to set it up in Datadog.
+
```
Comment (estherk15) [inline content/en/cloud_cost_management/aws.md:178]:
```suggestion
## Further reading

{{< partial name="whats-next/whats-next.html" >}}

```

File: content/en/cloud_cost_management/azure.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+title: Azure
+kind: documentation
+disable_toc: false
+private: true
```
Comment (estherk15) [inline content/en/cloud_cost_management/azure.md:5]: Same here, did you intend this to be private?

File: content/en/cloud_cost_management/azure.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
```
Comment (estherk15) [inline content/en/cloud_cost_management/azure.md:7]:
```suggestion
- link: "/cloud_cost_management/"
```

File: content/en/cloud_cost_management/aws.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
+---
+title: AWS
+kind: documentation
+disable_toc: false
+private: true
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
```
Comment (estherk15) [inline content/en/cloud_cost_management/aws.md:7]:
```suggestion
- link: "/cloud_cost_management/"
```

File: content/en/cloud_cost_management/_index.md [hunk 5]
```diff
@@ -29,262 +29,22 @@ Cloud Cost Management provides insights for engineering and finance teams to see
...
+  {{< nextlink href="/cloud_cost_management/azure">}}<u>Azure</u>: Configure Cloud Cost Management for your Azure bill. {{< /nextlink >}}
+ {{< /whatsnext >}}
 
 ## Cloud costs in dashboards
 
 Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
 
 {{< img src="cloud_cost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation" >}}
 
+Customers can export timeseries costs data using the [Metrics API][1]
```
Comment (estherk15) [inline content/en/cloud_cost_management/_index.md:44]:
```suggestion
```

File: content/en/cloud_cost_management/_index.md [hunk 6]
```diff
@@ -29,262 +29,22 @@ Cloud Cost Management provides insights for engineering and finance teams to see
...
-{{% /tab %}}
-{{< /tabs >}}
+{{< whatsnext desc="Get started with Cloud Cost Management:">}}
+  {{< nextlink href="/cloud_cost_management/aws">}}<u>AWS</u>: Configure Cloud Cost Management for your AWS bill.{{< /nextlink >}}
+  {{< nextlink href="/cloud_cost_management/azure">}}<u>Azure</u>: Configure Cloud Cost Management for your Azure bill. {{< /nextlink >}}
+ {{< /whatsnext >}}
 
 ## Cloud costs in dashboards
 
 Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
```
Comment (estherk15) [inline content/en/cloud_cost_management/_index.md:40]:
```suggestion
Visualize infrastructure spend alongside related utilization metrics to spot potential inefficiencies and savings opportunities:
- Add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source. 
- Export timeseries costs data using the [Metrics API][1].

```

File: content/en/cloud_cost_management/_index.md [hunk 7]
```diff
@@ -29,262 +29,22 @@ Cloud Cost Management provides insights for engineering and finance teams to see
...
 
 {{< img src="cloud_cost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation" >}}
 
+Customers can export timeseries costs data using the [Metrics API][1]
+
+[1]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-data-across-multiple-products
```
Comment (estherk15) [inline content/en/cloud_cost_management/_index.md:46]: Move this below the `Further Reading` section
```suggestion
```

File: content/en/cloud_cost_management/_index.md [hunk 8]
```diff
@@ -29,262 +29,22 @@ Cloud Cost Management provides insights for engineering and finance teams to see
...
+
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
```
Comment (estherk15) [inline content/en/cloud_cost_management/_index.md:50]:
```suggestion
{{< partial name="whats-next/whats-next.html" >}}

[1]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-data-across-multiple-products

```

File: content/en/cloud_cost_management/aws.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/azure"
+  tag: "Documentation"
+  text: "Gain insights in your Azure bill"
+---
+
+{{< jqmath-vanilla >}}
```
Comment (estherk15) [inline content/en/cloud_cost_management/aws.md:15]: I didn't see any formulas being used and removed.
```suggestion
```

File: content/en/cloud_cost_management/azure.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/aws"
+  tag: "Documentation"
+  text: "Gain insights in your AWS bill"
+---
+
+{{< jqmath-vanilla >}}
```
Comment (estherk15) [inline content/en/cloud_cost_management/azure.md:15]: Didn't see any formula used and removed.
```suggestion
```

**Review by estherk15 — COMMENTED** [review]
@klara-kr do you want to add these pages to the left navigation under **Cloud Cost** but above **Tag Pipelines**?

**Pull request merged**